### PR TITLE
Add TsfileSize column to 'show regions'

### PR DESCRIPTION
**Pull request by liyuheng55555**
image

### CODE REVIEW

File: iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/execution/config/metadata/ShowRegionTask.java [hunk 1]
```diff
@@ -111,6 +112,19 @@ public static void buildTSBlock(
...
+            == TConsensusGroupType.DataRegion.ordinal()) {
+          if (regionInfo.getTsfileSize() != -1) {
+            regionSizeStr = FileUtils.humanReadableByteCountSI(regionInfo.getTsfileSize());
+          } else {
+            regionSizeStr = "Unknown";
+          }
+        }
+        builder
+            .getColumnBuilder(12)
+            .writeBinary(new Binary(regionSizeStr, TSFileConfig.STRING_CHARSET));
```
Comment (wangchao316) [inline iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/execution/config/metadata/ShowRegionTask.java:127]: whether can use BytesUtils.valueOf() ?            change string to byte

Comment (liyuheng55555) [reply iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/execution/config/metadata/ShowRegionTask.java:127]: good point

File: iotdb-core/datanode/src/main/java/org/apache/iotdb/db/service/metrics/file/TsFileMetrics.java [hunk 2]
```diff
@@ -433,6 +434,27 @@ private synchronized void updateRemainData(boolean seq) {
...
+    for (Map<String, Pair<Long, Gauge>> map : seqFileSizeMap.values()) {
+      for (Map.Entry<String, Pair<Long, Gauge>> regionSizeEntry : map.entrySet()) {
+        Integer regionId = Integer.parseInt(regionSizeEntry.getKey());
+        regionSizeMap.put(regionId, regionSizeEntry.getValue().getLeft());
+      }
+    }
+    for (Map<String, Pair<Long, Gauge>> map : unseqFileSizeMap.values()) {
+      for (Map.Entry<String, Pair<Long, Gauge>> regionSizeEntry : map.entrySet()) {
+        Integer regionId = Integer.parseInt(regionSizeEntry.getKey());
+        if (regionSizeMap.containsKey(regionId)) {
```
[the remark this answers is not in the document]
Comment (liyuheng55555) [reply iotdb-core/datanode/src/main/java/org/apache/iotdb/db/service/metrics/file/TsFileMetrics.java:448]: maybe you mean merge API ?

File: iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/schema/column/ColumnHeaderConstant.java [hunk 3]
```diff
@@ -161,6 +161,7 @@ private ColumnHeaderConstant() {
   public static final String START_TIME = "StartTime";
   public static final String ROLE = "Role";
   public static final String CREATE_TIME = "CreateTime";
+  public static final String TSFILE_SIZE = "TsfileSize";
```
[the remark this answers is not in the document]
Comment (liyuheng55555) [reply iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/schema/column/ColumnHeaderConstant.java:164]: sure

**Review by OneSizeFitsQuorum — APPROVED** [review]
LGTM

**Pull request merged**